src/h5web/visualizations/heatmap/hooks.ts

```diff
@@ -145,7 +138,7 @@ export function usePanZoom(
   useEffect(() => {
     // Move camera on resize to stay within mesh bounds
     moveCameraTo(camera.position.x, camera.position.y);
-  }, [camera, moveCameraTo, size]); // `size` is key here
+  }, [camera, moveCameraTo]);
```

Comment:
`moveCameraTo` actually changes when `size` changes, so we're good with not having `size` as a dependency here.

Reply:
What makes `moveCameraTo` change on resize ?

Reply:
It's created with `useCallback()` and `size` as one of the dependencies.